src/home/pam_systemd_home.c

```diff
@@ -659,11 +634,9 @@ static int acquire_home(
                                 return pam_bus_log_parse_error(handle, r);
 
                         acquired_fd = fcntl(fd, F_DUPFD_CLOEXEC, 3);
-                        if (acquired_fd < 0) {
-                                pam_syslog(handle, LOG_ERR, "Failed to duplicate acquired fd: %s", bus_error_message(&error, r));
-                                return PAM_SERVICE_ERR;
-                        }
-
+                        if (acquired_fd < 0)
+                                return pam_syslog_errno(handle, LOG_ERR, acquired_fd,
```

Comment:
`acquired_fd` -> `errno`

Reply:
Both issues fixed. (I think a similar issue in another place instead. There was more than one like this.)